packages/camera/camera_avfoundation/ios/camera_avfoundation/Sources/camera_avfoundation/include/camera_avfoundation/FLTFileWriting.h

```diff
@@ -0,0 +1,24 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+@import Foundation;
+
+NS_ASSUME_NONNULL_BEGIN
+
+/// A protocol which abstracts the file writing operation implemented in `NSData`.
+/// It exists to allow replacing `NSData` in tests.
+@protocol FLTFileWriting <NSObject>
```

Comment:
Maybe `FLTWritableData`? It's unintuitive to have the data provider return something that sounds like it's a utility for writing files rather than something that sounds like it actually contains data.

Reply:
Good idea, makes sense